### PR TITLE
Automatic commit: Move 'cp-workflow-2-create-module-cf', 'cp-workflow…

### PR DESCRIPTION
**Pull request by AstridHerbst**
…-3-add-usertask-cf', 'cp-workflow-4-approval-ui-cf' from QA to Production

### CODE REVIEW

File: tutorials/cp-workflow-4-approval-ui-cf/cp-workflow-4-approval-ui-cf.md [hunk 1]
```diff
@@ -1,118 +1,182 @@
...
 [ACCORDION-END]
 
 
 [ACCORDION-BEGIN [Step 3: ](Add fields to your start form)]
 
-1. In the form editor (for the ``RequestForm.form``), select the **FIELDS** section.
+Once the form editor opens, add form data by choosing **Add Field**.
 
-2. To add two new fields, click **Add Field** twice, and enter the following data to define the fields:
+These expressions refer to the attributes in the JSON file that we have created in the previous step.
```
Comment (MichaelCzcz) [inline tutorials/cp-workflow-4-approval-ui-cf/cp-workflow-4-approval-ui-cf.md:78]:
```suggestion
These expressions refer to the attributes in the JSON file that you have created in the previous step.
```

File: tutorials/cp-workflow-4-approval-ui-cf/cp-workflow-4-approval-ui-cf.md [hunk 2]
```diff
@@ -1,118 +1,182 @@
...
+    ![Import MTA](upload-files-mtar.png)
 
+    - In the **Import** dialog, browse for the `WorkflowManagementFLP.tar` file that you extracted in your local system and choose **Open**.
 
-[ACCORDION-BEGIN [Step 6: ](Configure the business application visualization for your custom tile)]
-1. In your workspace of SAP Web IDE under **`sample.workflowtiles.mta.trial`**, select the **`portal-site`** folder and right-click it.
+    - The multitarget application gets imported under the **PROJECTS** explorer.
 
-2. Choose **New** | **Folder**.
+    - Next, we need to extract the **tar** file. For this, open the new terminal window and enter the commands one-by-one to uncompress the uploaded file.
```
Comment (MichaelCzcz) [inline tutorials/cp-workflow-4-approval-ui-cf/cp-workflow-4-approval-ui-cf.md:152]:
```suggestion
    - Next, you need to extract the **tar** file. For this, open the new terminal window and enter the commands one-by-one to uncompress the uploaded file.
```

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please fix those two minor issues and sign CLA.

**Comment by AstridHerbst** [comment]
I have exchanged "we" with "you" now

File: tutorials/cp-workflow-3-add-usertask-cf/cp-workflow-3-add-usertask-cf.md [hunk 3]
```diff
@@ -1,168 +1,198 @@
...
+    ![Create Form File](create-approval-form.png)
 
-[DONE]
-[ACCORDION-END]
+3. Choose **Create**.
 
+4. Once the form editor opens, add form data by choosing **Add Field**.
 
-[ACCORDION-BEGIN [Step 3: ](Add fields to your task form)]
+    These expressions refer to the attributes in the JSON file that we have created in the previous step.
```
Comment (MichaelCzcz) [inline tutorials/cp-workflow-3-add-usertask-cf/cp-workflow-3-add-usertask-cf.md:60]: Missed this one the first time.

```suggestion
    These expressions refer to the attributes in the JSON file that you have created in the previous step.
```

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Pull request merged**